doc/dev/Track2TestFramework.md

```diff
@@ -256,4 +256,48 @@ You can use `if (Mode == RecordingMode.Playback) { ... }` to change behavior for
 
 You can use `using (Recording.DisableRecording()) { ... }` to disable recording in the code block (useful for polling methods)
 
+# Support multi service version testing
 
+To enable multi-version testing add the `ClientTestFixture` attribute listing all the service versions to the test class itself or a base class:
+
+```C#
+[ClientTestFixture(
+    BlobClientOptions.ServiceVersion.V2019_02_02,
+    BlobClientOptions.ServiceVersion.V2019_07_07)]
+public abstract class BlobTestBase : StorageTestBase
+{
+    private readonly BlobClientOptions.ServiceVersion _serviceVersion;
+
+    public BlobTestBase(bool async, BlobClientOptions.ServiceVersion serviceVersion, RecordedTestMode? mode = null)
+        : base(async, mode)
+    {
+        _serviceVersion = serviceVersion;
+    }
+
+    // ...
+}
+```
+
+Add a service version attribute to the test class ctor and use the provided service version to create the `ClientOptions` instance.
+
+```C#
+public BlobClientOptions GetOptions() =>
+    new BlobClientOptions(_serviceVersion) { /* ... */ };
+```
+
+To control what service versions test would run against use the `ServiceVersion` attribute by setting it's `Mix` or `Max` properties (inclusive).
+
+```C#
+[Test]
+[ServiceVersion(Min = BlobClientOptions.ServiceVersion.V2019_02_02)]
+public void Ctor_ConnectionString()
```

Comment:
Nit - let's replace `public void Ctor_ConnectionString()` with something like `public async Task UploadOverwritesExistingBlob()` to encourage better test naming patterns than we use in Storage today.